test/integration/host_test.rb

```diff
@@ -449,14 +449,14 @@ def test_show_action_buttons
       click_link 'Parameters'
       assert page.has_selector?('#inherited_parameters .btn[data-tag=override]')
       page.find('#inherited_parameters .btn[data-tag=override]').click
-      refute page.has_selector?('#inherited_parameters .btn[data-tag=override]')
+      assert page.has_no_selector?('#inherited_parameters .btn[data-tag=override]')
```

Comment:
https://github.com/jnicklas/capybara#asynchronous-javascript-ajax-and-friends says it does.